database.sql

```diff
@@ -0,0 +1,10616 @@
+  `encounter_id` bigint(20) NOT NULL DEFAULT '0' COMMENT 'Encounter id if tagged',
+  `encounter_check` TINYINT(1) NOT NULL DEFAULT '0' COMMENT 'If encounter is created while tagging',
+  `audit_master_approval_status` TINYINT NOT NULL DEFAULT 1 COMMENT 'approval_status from audit_master table',
+  `audit_master_id` int(11) default NULL,
+  `documentationOf` varchar(255) DEFAULT NULL,
+  `encrypted` TINYINT(4) NOT NULL DEFAULT '0' COMMENT '0->No,1->Yes',
+  PRIMARY KEY  (`id`),
+  KEY `revision` (`revision`),
+  KEY `foreign_id` (`foreign_id`),
+  KEY `owner` (`owner`)
+) ENGINE=InnoDB;
+
+-----------------------------------------------------------
+
+--
+-- Table structure for table `documents_legal_detail`
+--
+
+DROP TABLE IF EXISTS `documents_legal_detail`;
+CREATE TABLE `documents_legal_detail` (
+  `dld_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `dld_pid` int(10) unsigned DEFAULT NULL,
+  `dld_facility` int(10) unsigned DEFAULT NULL,
+  `dld_provider` int(10) unsigned DEFAULT NULL,
+  `dld_encounter` int(10) unsigned DEFAULT NULL,
+  `dld_master_docid` int(10) unsigned NOT NULL,
+  `dld_signed` smallint(5) unsigned NOT NULL COMMENT '0-Not Signed or Cannot Sign(Layout),1-Signed,2-Ready to sign,3-Denied(Pat Regi),4-Patient Upload,10-Save(Layout)',
+  `dld_signed_time` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  `dld_filepath` varchar(75) DEFAULT NULL,
+  `dld_filename` varchar(45) NOT NULL,
+  `dld_signing_person` varchar(50) NOT NULL,
+  `dld_sign_level` int(11) NOT NULL COMMENT 'Sign flow level',
+  `dld_content` varchar(50) NOT NULL COMMENT 'Layout sign position',
+  `dld_file_for_pdf_generation` blob NOT NULL COMMENT 'The filled details in the fdf file is stored here.Patient Registration Screen',
+  `dld_denial_reason` longtext,
+  `dld_moved` tinyint(4) NOT NULL DEFAULT '0',
+  `dld_patient_comments` text COMMENT 'Patient comments stored here',
+  PRIMARY KEY (`dld_id`)
+) ENGINE=InnoDB AUTO_INCREMENT=1 ;
+
+-----------------------------------------------------------
+
+--
+-- Table structure for table `documents_legal_master`
+--
+
+DROP TABLE IF EXISTS `documents_legal_master`;
+CREATE TABLE `documents_legal_master` (
+  `dlm_category` int(10) unsigned DEFAULT NULL,
+  `dlm_subcategory` int(10) unsigned DEFAULT NULL,
+  `dlm_document_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `dlm_document_name` varchar(75) NOT NULL,
+  `dlm_filepath` varchar(75) NOT NULL,
+  `dlm_facility` int(10) unsigned DEFAULT NULL,
+  `dlm_provider` int(10) unsigned DEFAULT NULL,
+  `dlm_sign_height` double NOT NULL,
+  `dlm_sign_width` double NOT NULL,
+  `dlm_filename` varchar(45) NOT NULL,
+  `dlm_effective_date` datetime NOT NULL,
+  `dlm_version` int(10) unsigned NOT NULL,
+  `content` varchar(255) NOT NULL,
+  `dlm_savedsign` varchar(255) DEFAULT NULL COMMENT '0-Yes 1-No',
+  `dlm_review` varchar(255) DEFAULT NULL COMMENT '0-Yes 1-No',
+  `dlm_upload_type` tinyint(4) DEFAULT '0' COMMENT '0-Provide
```

Comment:
Code Removal after line 4887:
``sql_version` VARCHAR(150) NOT NULL,
  `acl_version` VARCHAR(150) NOT NULL,`
Removed lines 4888 and 4889